_episodes/07-cond.md

```diff
@@ -98,6 +98,18 @@ else:
 Note that to test for equality we use a double equals sign `==`
 rather than a single equals sign `=` which is used to assign values.
 
+> ## Comparing in python
+>
+> We've already seen a few of these already, but there are a few more options that you should know about:
```

Comment:
The above sentence uses "already" a couple times in the first clause, what do you think about changing the wording slightly to something like:
"Along with the  `>` and `==` operators we have already used for comparing values in our conditionals, there are a few more options to know about:"